### PR TITLE
Add CLI support for client tags

### PR DESCRIPTION
**Pull request by raghavsethi**

### CODE REVIEW

File: presto-client/src/main/java/com/facebook/presto/client/StatementClient.java [hunk 1]
```diff
@@ -142,6 +144,9 @@ private Request buildQueryRequest(ClientSession session, String query)
         if (session.getSource() != null) {
             builder.addHeader(PRESTO_SOURCE, session.getSource());
         }
+        if (session.getSource() != null) {
```
Comment (AlekseiS) [inline presto-client/src/main/java/com/facebook/presto/client/StatementClient.java]: Likely, `getClientTags`

Comment (raghavsethi) [reply presto-client/src/main/java/com/facebook/presto/client/StatementClient.java]: Thanks! Fixed.

File: presto-client/src/main/java/com/facebook/presto/client/ClientSession.java [hunk 2]
```diff
@@ -152,6 +161,10 @@ public ClientSession(
         this.server = requireNonNull(server, "server is null");
         this.user = user;
         this.source = source;
+        for (String clientTag : requireNonNull(clientTags, "clientTags is null")) {
```
Comment (electrum) [inline presto-client/src/main/java/com/facebook/presto/client/ClientSession.java]: Let's move this below, like we do with the session property validation

File: presto-client/src/main/java/com/facebook/presto/client/ClientSession.java [hunk 3]
```diff
@@ -152,6 +161,10 @@ public ClientSession(
...
+            checkArgument(!clientTag.contains(","), "clientTag cannot contain ','");
```
Comment (electrum) [inline presto-client/src/main/java/com/facebook/presto/client/ClientSession.java]: This should be "clientTags"

**Pull request closed without merging**